diff -p1 st-master/config.def.h st/config.def.h
*** st-master/config.def.h	2022-03-28 20:27:16.954909487 -0700
--- st/config.def.h	2022-03-28 20:21:37.673918934 -0700
***************
*** 7,9 ****
   */
! static char *font = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
  static int borderpx = 2;
--- 7,9 ----
   */
! static char *font = "Roboto Mono:size=12:antialias=true:autohint=true";
  static int borderpx = 2;
*************** static const char *colorname[] = {
*** 98,117 ****
  	/* 8 normal colors */
! 	"black",
! 	"red3",
! 	"green3",
! 	"yellow3",
! 	"blue2",
! 	"magenta3",
! 	"cyan3",
! 	"gray90",
  
  	/* 8 bright colors */
! 	"gray50",
! 	"red",
! 	"green",
! 	"yellow",
! 	"#5c5cff",
! 	"magenta",
! 	"cyan",
! 	"white",
  
--- 98,117 ----
  	/* 8 normal colors */
! 	"#282a2e",  /* black */
! 	"#a54242",  /* red */
! 	"#8c9440",  /* green */
! 	"#de935f",  /* yellow */
! 	"#5f819d",  /* blue */
! 	"#85678f",  /* magenta */
! 	"#5e8d87",  /* cyan */
! 	"#707880",  /* gray */
  
  	/* 8 bright colors */
! 	"#373b41",  /* black */
! 	"#cc6666",  /* red */
! 	"#b5bd68",  /* green */
! 	"#f0c674",  /* yellow */
! 	"#81a2be",  /* blue */
! 	"#b294bb",  /* magenta */
! 	"#8abeb7",  /* cyan */
! 	"#c5c8c6",  /* gray */
  
*************** static const char *colorname[] = {
*** 122,125 ****
  	"#555555",
! 	"gray90", /* default foreground colour */
! 	"black", /* default background colour */
  };
--- 122,125 ----
  	"#555555",
! 	"#c5c8c6", /* default foreground colour */
! 	"#1a191d", /* default background colour */
  };
*************** static Shortcut shortcuts[] = {
*** 203,204 ****
--- 203,206 ----
  	{ TERMMOD,              XK_Num_Lock,    numlock,        {.i =  0} },
+ 	/* { TERMMOD,              XK_Return,      newterm,        {.i =  0} }, */
+ 	{ (Mod1Mask|ShiftMask), XK_Return,      newterm,        {.i =  0} },
  };
Common subdirectories: st-master/.git and st/.git
diff -p1 st-master/Makefile st/Makefile
*** st-master/Makefile	2022-03-28 20:27:16.954909487 -0700
--- st/Makefile	2022-03-28 19:40:42.549878071 -0700
*************** install: st
*** 51,52 ****
--- 51,54 ----
  	@echo Please see the README file regarding the terminfo entry of st.
+ 	mkdir -p $(DESTDIR)$(PREFIX)/share/applications
+ 	cp -f st.desktop $(DESTDIR)$(PREFIX)/share/applications
  
*************** uninstall:
*** 55,56 ****
--- 57,59 ----
  	rm -f $(DESTDIR)$(MANPREFIX)/man1/st.1
+ 	rm -f $(DESTDIR)$(PREFIX)/share/applications/st.desktop
  
diff -p1 st-master/st.c st/st.c
*** st-master/st.c	2022-03-28 20:27:16.954909487 -0700
--- st/st.c	2022-03-28 19:53:07.323246078 -0700
*************** typedef struct {
*** 155,156 ****
--- 155,157 ----
  static void execsh(char *, char **);
+ static char *getcwd_by_pid(pid_t pid);
  static void stty(char **);
*************** tswapscreen(void)
*** 1054,1055 ****
--- 1055,1087 ----
  }
+ 
+ void
+ newterm(const Arg* a)
+ {
+ 	int res;
+ 	switch (fork()) {
+ 	case -1:
+ 		die("fork failed: %s\n", strerror(errno));
+ 		break;
+ 	case 0:
+ 		switch (fork()) {
+ 		case -1:
+ 			die("fork failed: %s\n", strerror(errno));
+ 			break;
+ 		case 0:
+ 			res = chdir(getcwd_by_pid(pid));
+ 			execlp("st", "./st", NULL);
+ 			break;
+ 		default:
+ 			exit(0);
+ 		}
+ 	default:
+ 		wait(NULL);
+ 	}
+ }
+ 
+ static char *getcwd_by_pid(pid_t pid) {
+ 	char buf[32];
+ 	snprintf(buf, sizeof buf, "/proc/%d/cwd", pid);
+ 	return realpath(buf, NULL);
+ }
  
Only in st: st.desktop
diff -p1 st-master/st.h st/st.h
*** st-master/st.h	2022-03-28 20:27:16.954909487 -0700
--- st/st.h	2022-03-28 19:53:07.323246078 -0700
*************** void draw(void);
*** 83,84 ****
--- 83,85 ----
  
+ void newterm(const Arg *);
  void printscreen(const Arg *);
