Common subdirectories: dmenu/.git and dmenu_barpad/.git
diff -Np1 dmenu/config.def.h dmenu_barpad/config.def.h
*** dmenu/config.def.h	2022-02-12 15:22:11.791931544 -0800
--- dmenu_barpad/config.def.h	2022-02-12 15:19:38.145065813 -0800
***************
*** 4,5 ****
--- 4,6 ----
  static int topbar = 1;                      /* -b  option; if 0, dmenu appears at bottom     */
+ static const unsigned int bar_hpad  = 6;    /* default 2px above/below font size  */
  /* -fn option overrides fonts[0]; default X11 font or font set */
Binary files dmenu/dmenu and dmenu_barpad/dmenu differ
diff -Np1 dmenu/dmenu.c dmenu_barpad/dmenu.c
*** dmenu/dmenu.c	2022-02-12 15:22:11.791931544 -0800
--- dmenu_barpad/dmenu.c	2022-02-12 15:20:00.285334662 -0800
*************** setup(void)
*** 628,630 ****
  	/* calculate menu geometry */
! 	bh = drw->fonts->h + 2;
  	lines = MAX(lines, 0);
--- 628,630 ----
  	/* calculate menu geometry */
! 	bh = drw->fonts->h + bar_hpad;
  	lines = MAX(lines, 0);
Binary files dmenu/dmenu.o and dmenu_barpad/dmenu.o differ
Binary files dmenu/drw.o and dmenu_barpad/drw.o differ
Binary files dmenu/stest and dmenu_barpad/stest differ
Binary files dmenu/stest.o and dmenu_barpad/stest.o differ
Binary files dmenu/util.o and dmenu_barpad/util.o differ
